### ISSUE TITLE
Unknown plugin "rewire"

### ISSUE THREAD

**Issue by daralthus** [wallabyjs/wallaby-rewire#1]
Hey guys and ides on this? After a fresh clone and npm install I get:

```
Failed to run compilers on src/a.jsx, ReferenceError: Unknown plugin "rewire" at        PluginManager.subnormaliseString (/Users/peterkadlot/Documents/prgmr/wallaby-rewire/node_modules/babel-core/lib/transformation/file/plugin-manager.js:147:13) at PluginManager.add (/Users/peterkadlot/Documents/prgmr/wallaby-rewire/node_modules/babel-core/lib/transformation/file/plugin-manager.js:190:40) at File.buildTransformers (/Users/peterkadlot/Documents/prgmr/wallaby-rewire/node_modules/babel-core/lib/transformation/file/index.js:237:21) at new File (/Users/peterkadlot/Documents/prgmr/wallaby-rewire/node_modules/babel-core/lib/transformation/file/index.js:139:10) at Pipeline.transform (/Users/peterkadlot/Documents/prgmr/wallaby-rewire/node_modules/babel-core/lib/transformation/pipeline.js:164:16) at Array.forEach (native) at emitTwo (events.js:87:13) at process.emit (events.js:169:7) at handleMessage (child_process.js:306:10) at Pipe.channel.onread (child_process.js:334:11)
```

I tried `babel-plugin-rewire` as well, but same issue.

**Comment by ArtemGovorov** [comment]
Can't reproduce this one. What OS and wallaby integration (IntelliJ platform, Visual Studio, Atom) do you use? Could you please also check that you have the `babel-plugin-rewire` in your node modules folder.

**Comment by daralthus** [comment]
Thanks for looking into this!

I use:
Atom 1.0.8 
OSX 10.10.01
Tested with node 0.10.22 and iojs 3.2.0

The babel-plugin-rewire is in the node_modules.

I used babel with rewire in another project without wallaby and it picks it up both from the .babelrc and as a cli argument.

In the meantime I found a workaround in the babel docs so 

``` js
wallaby.compilers.babel({ babel: require('babel'), plugins: [require('babel-plugin-rewire)] })
```

works fine, but it would be nice to be able to use my .babelrc like 

``` js
wallaby.compilers.babel({ babel: require('babel'), babelrc: __dirname + './.babelrc' })
```

which is now unfortunately broken.

**Comment by ArtemGovorov** [comment]
Awesome, thanks for the update. We do have [the `.babelrc` support feature request](https://github.com/wallabyjs/public/issues/208), so I'll close this one. You may also find a way to use current `.babelrc` in the issue comments:

``` JavaScript
var babel = require('babel');

var babelConfig = JSON.parse(require('fs').readFileSync(require('path').join(__dirname, '.babelrc')));
babelConfig.babel = babel;

module.exports = function (wallaby) {
  return {
    files: [...],
    tests: [...],
    compilers: {
      '**/*.js': wallaby.compilers.babel(babelConfig)
    }
  };
};
```

**Comment by ArtemGovorov** [comment]
P.S. Also changed the repo to load plugin the way you did. But I'll have to have a look why it doesn't work the way it was in Atom (it works fine in WebStorm).

**Comment by ArtemGovorov** [comment]
Found and fixed the issue (Package 1.0.3, Plugin v1.0.1, Core v1.0.92), so now it should be possible to reference the plugin by its name, like:

``` JavaScript
wallaby.compilers.babel({ babel: require('babel'), plugins: ['rewire'] })
```
